tests/system/Config/FactoriesTest.php

```diff
@@ -0,0 +1,264 @@
+<?php
+
+namespace CodeIgniter\Config;
+
+use CodeIgniter\Config\Factories;
+use CodeIgniter\Test\CIUnitTestCase;
+use Tests\Support\Widgets\OtherWidget;
+use Tests\Support\Widgets\SomeWidget;
+use ReflectionClass;
+use stdClass;
+
+class FactoriesTest extends CIUnitTestCase
+{
+	protected function setUp(): void
+	{
+		parent::setUp();
+
+		Factories::reset();
+	}
+
+	protected function getFactoriesStaticProperty(...$params)
+	{
+		// First parameter is the actual property
+		$name = array_shift($params);
+
+		$factory  = new ReflectionClass(Factories::class);
+		$property = $factory->getStaticPropertyValue($name);
```

Comment:
This is a permission issue on lower PHP versions. This way, we should be able to get rid of test skipping.
```suggestion
		$factory  = new ReflectionClass(Factories::class);
		$properties = $factory->getStaticProperties();
		$property = $properties[$name] ?? [];
```